external/common/include/fast_cpp_csv_parser/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2015, ben-strasser
```

Comment:
I think `external/common/includes/CSV_README.md` would be a better place that `examples/` What do you think?